### PR TITLE
Creating a bio for ntend

### LINKED ISSUE #61: Add bio for ntend
## Test

** Favorite variable names
- [ ] foo
- [ ] bar
- [ ] baz

@githubstudent

### PR DESCRIPTION
**Pull request by ntend**
Resolves #61

Adding a bio for ntend.

@githubstudent  - please review.

### CODE REVIEW

File: bios/ntend-bio.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## ntend
+
+- Item 1
+- Item 2
+- Go bears!
```
Comment (johnboy55) [inline bios/ntend-bio.md:5]: Nit: Should be spelt cardinals wrong here :godmode:

Comment (sh-mikem) [inline bios/ntend-bio.md:5]: Favorite player?  (Football/basketball, past/present)

Comment (ntend) [inline bios/ntend-bio.md:5]: @johnboy55 Make like a tree and leaf

Comment (bkerrigan) [inline bios/ntend-bio.md:5]: Da :maple_leaf:

**Comment by sh-mikem** [comment]
Looks good!

**Pull request merged**